### DISCUSSION TITLE
The new video UX is very non-intuitive

### DISCUSSION THREAD

**Discussion by indicozy** [Docile-Alligator/Infinity-For-Reddit#809] [Completed Ideas] [answered]
Hi there! I love Infinity and I was using for about six months. Thanks for your project!

However, the new interaction design with video is making me crazy. Before the update, the first click showed the buttons and UI of the video and the second click hides all the buttons of the video. Now, the second click opens the video at full screen and it is very non-intuitive for me and really stresses me out.

I intuitively click on every video to see the video length and close the video UI by clicking again but now it really worsens my user experience.

Can we roll back to the previous UX or is it possible to create a toggle in settings to customize on your own?

**Comment by gabey14** [comment]
True I also have the same problem it would be better if there was an option to use the previous method.

**Comment by Wladefant** [comment, marked as answer]
There is an option to change back to the previous video ui: settings>video>legacy video controller ui

**Comment by indicozy** [comment]
I've turned it on and manually stopped application, though nothing
has changed.

**Comment by indicozy** [comment]
Okay, I've found that it still open full screen on touch if the video is
not playing yet. If it's playing, you can touch to open controls. It's a
bit unintuitive for me, but it's a matter of subjectivity.

**Reply by Wladefant** [reply]
Yes sorry we can't make the app for everyone perfect 😟

**Comment by indicozy** [comment]
No problem, it's a not an issue now, I'll close the issue later. Thanks for
reply!

**Reply by Wladefant** [reply]
That is not an issue anymore but rather a discussion so you don't have to close it 😉
